Automattic/jetpack
# Social: Hide share status when no connection is enabled

Currently, even if all the connections are disabled, we still show, `"Sharing to your social media…"` message, which is not ideal.

We don't need to show that status if all the connections are disabled.

This PR uses the hook added in #39119.

## Proposed changes:

* Hide the share status in the sidebar when all the connections are turned off.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:







* Enable the share status feature - `define( 'JETPACK_SOCIAL_HAS_SHARE_STATUS', true );`
* Publish a post to share it to a few accounts
* Confirm that the post publish share status panel shows the status
* Create another post with all the social connections disabled
* Publish the post
* Confirm that the share status panel is not displayed.
